gdk/src/keys.rs

```diff
@@ -79,6 +92,19 @@ impl Key {
         skip_assert_initialized!();
         unsafe { ffi::gdk_keyval_to_lower(**self) }.into()
     }
+
+    #[doc(alias = "gdk_keyval_convert_case")]
+    pub fn keyval_convert_case(&self) -> (Self, Self) {
```

Comment:
Maybe rename this one to just convert_case?

Reply:
Ah right, forgot to rename that.